bodhi/tests/server/test_metadata.py

```diff
@@ -24,6 +24,8 @@
 import os
 import shutil
 import tempfile
+import mock
+import pytest
```

Comment:
```pytest``` is a third party library, so should be grouped with ```createrepo_c``` below.

Reply:
https://github.com/mcurlej/bodhi/pull/1